scripts/deploy-erc20-tokens.ts

```diff
@@ -0,0 +1,97 @@
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+import { ethers } from "hardhat";
+import * as fs from "fs";
+
+async function main() {
+  let addresses = {};
+
+  // deploy UZHETH tokens
+  const UzhUniToken = await ethers.getContractFactory("UzhUniToken");
+  const uzhuni = await UzhUniToken.deploy();
+  await uzhuni.deployed();
+  console.log("Deployed UzhUniToken (UZHUNI): ", uzhuni.address);
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  addresses = { ...addresses, uzhuni: { address: uzhuni.address } };
```

Comment:
It's not React :D You can just use addresses['uzhuni']=... , no need to always spread and create a new object.